modules/core/src/main/scala/org/scalasteward/core/vcs/data/BuildRoot.scala

```diff
@@ -0,0 +1,19 @@
+/*
+ * Copyright 2018-2021 Scala Steward contributors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.scalasteward.core.vcs.data
+
+final case class BuildRoot(repo: Repo, relativeBuildRootPath: String)
```

Comment:
I suggest to rename `relativeBuildRootPath` to just `relativePath`. I don't think repeating the class name there adds more information.

Reply:
Done